app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::ApiController < OpenStax::Api::V1::ApiController
+  skip_before_filter :authenticate_user!
```

Comment:
It used to be in openstax_api, it got removed here: https://github.com/openstax/openstax_api/commit/643f4b5f2f0fa013f84f6c0808008b330393af50

Similarly for redirect_to.